src/bin/pgcopydb/pgcmd.c

```diff
@@ -977,16 +978,8 @@ pg_restore_db(PostgresPaths *pgPaths,
 		args[argsIndex++] = nspname;
 	}
 
-	if (listFilename != NULL)
-	{
-		args[argsIndex++] = "--use-list";
-		args[argsIndex++] = (char *) listFilename;
-	}
-
 	args[argsIndex++] = (char *) dumpFilename;
 
-	args[argsIndex] = NULL;
-
```

Comment:
Same as before, can we please keep that?

Reply:
the new change initializes the array with NULL values.